tests/verify-role-failed.yml

```diff
@@ -82,11 +82,14 @@
     - name: Verify return code and message is correct
       assert:
         that:
-          - snapshot_cmd.failed
-          - snapshot_cmd.rc != 0
-          - snapshot_cmd.stdout_lines is search(__snapshot_failed_regex) or
-            snapshot_cmd.stderr_lines is search(__snapshot_failed_regex)
+          - snapshot_cmd_raw is failed
+          - snapshot_cmd["return_code"] != 0
+          - snapshot_cmd["errors"] is search(__snapshot_failed_regex)
         msg: "{{ __snapshot_failed_msg }}"
```

Comment:
```suggestion
        var: __snapshot_failed_msg
```

Reply:
Looks like `var` is not a valid parameter for `assert`
```
    _VALID_ARGS = frozenset(('fail_msg', 'msg', 'quiet', 'success_msg', 'that'))
```
`msg` is an alias for `fail_msg`